api/rooms/views/detail.py

```diff
@@ -0,0 +1,51 @@
+from optparse import TitledHelpFormatter
+from django.http  import JsonResponse
+from django.views import View
+
+from ..models import Room, OptionType
+
+class RoomDetailView(View):
```

Comment:
Detail page 보다는 `Room`가 더 깔끔해보이는데 어떤가요? ㅎㅎ

```suggestion
class RoomView(View):
```

Reply:
그런것 같아요 ㅎㅎㅎ 수정하도록 하겠습니다 !